api/src/certification/evaluation/domain/models/ComplementaryCertificationCourse.js

```diff
@@ -0,0 +1,8 @@
+class ComplementaryCertificationCourse {
+  constructor({ complementaryCertificationKey, hasComplementaryReferential } = {}) {
+    this.complementaryCertificationKey = complementaryCertificationKey;
+    this.hasComplementaryReferential = hasComplementaryReferential;
```

Comment:
Si je mets de côté le contexte particulier que je connais, en première lecture, il me paraît bizarre qu'un `ComplementaryCertificationCourse` puisse avoir autre chose que `this.hasComplementaryReferential = true`.

Je comprends bien qu'il s'agit ici de la double certification dans le cas où `this.hasComplementaryReferential = false`, mais tout ça me fait penser qu'il serait peut-être  pertinent (et complémentaire ?) de faire émerger cette notion de `DoubleCertification`